documentation-website/Writerside/topics/Contributing.md

```diff
@@ -43,21 +43,29 @@ docker context list
 
 ### Pull Requests
 
-Contributions are made using Github [pull requests](https://help.github.com/en/articles/about-pull-requests):
+Contributions are made using GitHub [pull requests](https://help.github.com/en/articles/about-pull-requests):
 
-1. Fork the Exposed repository, because imitation is the sincerest form of flattery.
+1. Fork the [Exposed repository](https://github.com/JetBrains/Exposed), because imitation is the sincerest form of flattery.
 2. Clone your fork to your local machine.
 3. Create a new branch for your changes.
-4. [Create](https://github.com/JetBrains/Exposed/compare) a new PR with a request to merge to the **master** branch.
-5. Ensure that the description is clear and refers to an existing ticket/bug if applicable, prefixing the description with
-   EXPOSED-{NUM}, where {NUM} refers to the YouTrack issue.
+4. [Create](https://github.com/JetBrains/Exposed/compare) a new PR with a request to merge to the **main** branch.
+5. Ensure that the PR title is clear and refers to an [existing ticket/bug](https://youtrack.jetbrains.com/issues/EXPOSED) if applicable,
+   prefixing the title with both a [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary)
+   and EXPOSED-\{NUM\}, where \{NUM\} refers to the YouTrack issue code.
```

Comment:
In the Writerside preview and on the [website](http://jetbrains.github.io/Exposed/contributing.html#pull-requests), `{NUM}` as it was before will not appear.
I thought escaping `\{NUM\}` would help but it renders without change as `\{NUM\}`.
@vnikolova Do you know how to get just `{NUM}` in markdown?
I could just switch to `[NUM]`, which seems to render properly without needing any escape. Or maybe something like `EXPOSED-###` would be more descriptive?

Reply:
HTML entity works 🙌